crates/bifrost/src/appender.rs

```diff
@@ -119,8 +119,28 @@ impl Appender {
 
                     self.loglet_cache = Some(new_loglet);
                 }
-                Err(AppendError::Shutdown(e)) => return Err(Error::Shutdown(e)),
-                Err(AppendError::Other(e)) => return Err(Error::LogletError(e)),
+                Err(AppendError::Other(err)) if err.retryable() => {
```

Comment:
I really think this is what is causing issues with the remote sequencer. From what I saw in the logs so far is that sometimes when connection is closed, the commit is resolved with a retryable error. Hence the caller (the Appender) in this case need to retry again